app/views/layouts/projects.html.erb

```diff
@@ -32,13 +32,20 @@
             ) %>
           <% end %>
           <% if allowed_to?(:update?, @project) %>
-            <%= render section.with_item(
-              url: project_edit_path(@project),
-              icon: "cog_6_tooth",
-              label: t(:"projects.sidebar.settings"),
-              selected: @current_page == t(:"projects.sidebar.settings").downcase
-            ) %>
+            <%= render section.with_dropdown(label: t(:"projects.sidebar.settings") , icon: "cog_6_tooth", caret: true, skidding: 200, distance: -45, dropdown_position: :right, selected: (@current_page == t(:"projects.sidebar.settings").downcase) || (@current_page == t(:"projects.sidebar.bot_accounts").downcase), trigger: :hover,
```

Comment:
**suggestion:** Try overriding the focus ring CSS on the dropdown button, that should get rid of that weird state when `Settings` is clicked on.

Reply:
Focus ring overridden in ce495a0c97238b35dcc0a04e8e7f5f564ec876e0